server/cluster/table_manager_test.go

```diff
@@ -191,7 +191,12 @@ func testCreateTable(ctx context.Context, re *require.Assertions, manager Manage
 ) {
 	c, err := manager.GetCluster(ctx, clusterName)
 	re.NoError(err)
-	table, err := c.CreateTable(ctx, shardID, schema, tableName, false)
+	table, err := c.CreateTable(ctx, CreateTableRequest{
+		ShardID:       shardID,
+		SchemaName:    schema,
+		TableName:     tableName,
+		PartitionInfo: storage.PartitionInfo{},
```

Comment:
Remove it, `table_manager_test` used to test normal table, `PartitionInfo` should be nil.
If you want to test create partition table in cluster, you should define the new function`testCreatePartitionTable`.

Reply:
`storage.PartitionInfo{}` means the table is not a `PartitionTable`.